src/main/java/uk/gov/ch/oauth/Oauth2.java

```diff
@@ -201,4 +202,26 @@ private void regenerateSessionID(HttpServletResponse httpServletResponse) {
 
         httpServletResponse.addCookie(buildSessionCookie(session));
     }
+
+    public void invalidateSession(Session chSession, String signInfo, Store store){
+        final Map<String, Object> sessionData = chSession.getData();
+        if (chSession.getSignInInfo().isSignedIn()){
+            removeSessionInfo(sessionData, signInfo);
+            removeZXSInfo(sessionData, store);
+        }
+    }
```

Comment:
If this is public could you setup Javadocs.

Reply:
Ok, I may need you to remind me how because I have forgotten and the interwebs is not being very helpful

Reply:
They look like this 
/**
  *
  */
IntelliJ is good at putting the things inbetween so just do the first bit then hit enter

Reply:
Done